website/content/docs/integrations/consul-integration.mdx

```diff
@@ -111,134 +309,174 @@ remaining fields are required to match those shown here.
-there is no way to provide Nomad certificates to Consul (or Vault) clients. If
-you want to use mTLS, which is the recommended practice, you need to terminate
-TLS before the requests reach Nomad JWKS endpoint.
+<CodeBlockConfig highlight="6-9" filename="auth-method.json">
 
-</Note>
+```json
+{
+  "JWKSURL": "http://nomad.example.com:4646/.well-known/jwks.json",
+  "JWTSupportedAlgs": ["RS256"],
+  "BoundAudiences": ["consul.io"],
+  "ClaimMappings": {
+    "nomad_namespace": "nomad_namespace",
+    "nomad_job_id": "nomad_job_id",
+    "nomad_task": "nomad_task",
+    "nomad_service": "nomad_service"
+  }
+}
+```
 
-Using that configuration file, you'll create a Consul Auth Method named
-`nomad-workloads`, which controls:
+</CodeBlockConfig>
 
-* Authentication for Service Identity tokens used to register services and
-  configure Consul Connect.
+##### Consul Binding Rules
 
-* Access to reading service data and Consul KV for your jobs' `template`
-  blocks.
+Consul auth methods use [binding rules][consul_binding_rules] to determine the
+set of policies applied to the generated ACL token. Nomad workload identities
+can be used in Consul for two main purpose: registering services and retrieving
+configuration values from Consul KV using `template` blocks in tasks. Each
+purpose requires tokens with different permissions so Nomad requires two
+binding rules.
 
-The auth method will have two binding rules attached: one mapping Nomad
-`service` blocks to Consul services, and one mapping Nomad namespaces or your
-job IDs to Consul Roles.
+The first binding rule associates the Consul ACL token with a [service
+identity][consul_si], allowing the token to register and manage the lifecyle of
+a given service. This binding rule is only applied to Nomad workload identities
+for services because they are the only ones that have a `nomad_service` claim.
 
-<Note>
+```shell-session
+$ consul acl binding-rule create \
+    -method 'nomad-workloads' \
+    -bind-type 'service' \
+    -bind-name '${value.nomad_service}' \
+    -selector '"nomad_service" in value'
+```
 
-Nomad also supports having separate auth methods for services and tasks, for
-additional flexibility. Consult [`service_auth_method`] and [`task_auth_method`]
-configuration parameters documentation.
+The [`-bind-name`][] flag restricts the token to only be able to modify
+services with the same name as the one defined in the Nomad workload identity
+claim. The [`-selector`][] flag ensures this binding rule only applies to
+workload identities for services.
 
-</Note>
+The second binding rule associates the Consul ACL token with an [ACL
+role][consul_acl_role], which is a collection of [ACL
+policies][consul_acl_policy] that define what the token is authorized to do.
+This binding rule is applied to Nomad workload identities for tasks to access
+information from Consul. The exact ACL policy rules will depend on the level of
+access required by the tasks.
```

Comment:
Maybe add a parenthetical here like "(typically, accessing service addresses and KV with `template` blocks)"?